src/target/linux.rs

```diff
@@ -417,7 +417,14 @@ mod tests {
 
         let threads: Vec<_> = threads
             .iter()
-            .map(|t| (t.name().unwrap().unwrap().clone(), t.thread_id()))
+            .map(|t| {
+                let name = match t.name() {
+                    Ok(Some(name)) => name,
+                    Ok(None) => "".to_string(),
+                    Err(err) => err.to_string(),
```

Comment:
I think only `Ok(Some(_))` and `Ok(None)` can happen by chance without there actually being a problem in either headcrab or the kernel. I think `Err(_)` would indicate a bug.

Reply:
I see. Do you think I should just unwrap the result without worries?

Reply:
Yes

Reply:
Thank you! Fixed.